src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -702,7 +703,8 @@ subroutine SoilStateInitTimeConst(bounds, soilstate_inst, nlfilename)
     do c = begc,endc
        g = col%gridcell(c)
 
-       soilstate_inst%gwc_thr_col(c) = 0.17_r8 + 0.14_r8 * clay3d(g,1) * 0.01_r8
+       soilstate_inst%gwc_thr_col(c) = 0.01_r8*(0.17_r8*clay3d(g,1) + 0.0014_r8*clay3d(g,1)*clay3d(g,1))  ! Danny M. Leung modified the equation of soil moisture effect for dust emissions using a scale factor of 1. 0.01 is to convert from % to fraction.
```

Comment:
This preexisted before, but some of these should be parameters. This probably should be made into an issue. 0.17, 0.01, 0.0014 etc.

Reply:
I'll make into an issue since this preexisted.

Reply:
These are fitting parameters in a regression by Fecan 1999. I think it's easier to comment on them in the same line.

Reply:
Since, this is pretty restrained to the functions I'm not going to worry about this for now. A search for 0.17 showed that there are some instances where the 0.17 soil layer is hardcoded in, and then these Fecan parameters here and that's it.